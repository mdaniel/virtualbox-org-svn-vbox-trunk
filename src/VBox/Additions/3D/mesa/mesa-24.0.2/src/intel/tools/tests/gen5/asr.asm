asr.nz.f0.0(8)  null<1>D        -g1.6<0,1,0>D   31D             { align1 };
asr.nz.f0.0(16) null<1>D        -g1.6<0,1,0>D   31D             { align1 compr };
asr(8)          g4<1>D          g5<4>D          g4<4>UD         { align16 };
asr(8)          g11<1>.xD       g5<4>.xD        0x00000002UD    { align16 };
asr(8)          g5<1>D          g3<8,8,1>D      0x00000002UD    { align1 };
asr(16)         g10<1>D         g6<8,8,1>D      0x00000002UD    { align1 compr };
