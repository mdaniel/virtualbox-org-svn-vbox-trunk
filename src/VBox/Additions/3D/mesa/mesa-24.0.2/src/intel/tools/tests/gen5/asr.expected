0c 00 60 02 a4 1c 00 20 38 40 00 00 1f 00 00 00
0c 20 80 02 a4 1c 00 20 38 40 00 00 1f 00 00 00
0c 01 60 00 a5 04 8f 20 a4 00 6e 00 84 00 6e 00
0c 01 60 00 a5 0c 61 21 a0 00 60 00 02 00 00 00
0c 00 60 00 a5 0c a0 20 60 00 8d 00 02 00 00 00
0c 20 80 00 a5 0c 40 21 c0 00 8d 00 02 00 00 00
